src/lib/style/components/_upset-plot.scss

```diff
@@ -1,12 +1,45 @@
 .UpsetPlot {
-  // select any <line> element that has a class name that starts with "upsetLine-upset-".  
-  // Upset.js authors use a unique class name (React.useMemo + random number generator), which makes it more difficult to target.
   line{
     stroke-width: 0;
     stroke-opacity: 0;
   }
+  // Upset.js authors use a unique class name (React.useMemo + random number generator), which makes it more difficult to target.
+  // select any <text> element that has a class name that starts with "cBarTextStyle-upset-".  
+  text[class^='cBarTextStyle-upset-'], text[class*=' cBarTextStyle-upset-']{
+    display: none;
+  }
+  text[class^='sBarTextStyle-upset-'], text[class*=' sBarTextStyle-upset-']{
+    display: none;
+  }
+  circle[class^='fillNotMember-upset-'], text[class*=' fillNotMember-upset-']{
+    stroke: #d3d3d3;
+    stroke-width: 1;
+  }
+  
+  // rotate axis label (so that it's vertical), and it move down by 12%
+  text[class^='cChartTextStyle-upset-'], text[class*=' cChartTextStyle-upset-']{
+    text-transform: uppercase;
+  }
+  
+  // rotate axis label (so that it's vertical), and it move down by 12%
+  text[class^='sChartTextStyle-upset-'], text[class*=' sChartTextStyle-upset-']{
+    transform: rotate(270deg) translateX(-12%);
+    text-transform: uppercase;
```

Comment:
text-transform doesn't seem to be available for text tag. See https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text. Might want to double check this one. Otherwise, everything else looks okay to me.

Reply:
Nice catch!  It seems to work on Chrome and FF (79), but perhaps this doesn't work on other browser versions.  Changed to txt.toUpperCase() in the js instead.